MC/Geant4Config.C

```diff
@@ -18,9 +18,9 @@ void Geant4Config()
   TGeant4 *geant4 = 0;
   if ( ! gMC ) {
     TG4RunConfiguration* runConfiguration = NULL;
-    TString configPhysList = "FTFP_INCLXX_EMV+optical";
+    TString configPhysList = "FTFP_BERT_EMV+optical+biasing";
     if (g4PhysList == "BERT") configPhysList = "FTFP_BERT_EMV+optical";
-    else if (g4PhysList == "BERT+biasing") configPhysList = "FTFP_BERT_EMV+optical+biasing";
```

Comment:
I think that this line can stay so that you have always the three options regardless what the default is:
BERT = FTFP_INCLXX_EMV+optical
BERT+biasing = FTFP_BERT_EMV+optical+biasing
INCLXX = FTFP_INCLXX_EMV+optical

Reply:
Done, I added also a further check.